app/Http/Controllers/Admin/PostController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller as BaseController;
+use Illuminate\Http\Request;
+use App\Models\Post;
+use App\Models\Category;
+use Illuminate\Support\Facades\Auth;
+
+class PostController extends BaseController
+{
+	public function create()
+	{
+		$categories = Category::all();
+		
+		return view('admin.post.create', compact('categories'));
+	}
+	public function store(Request $request)
+	{
+		// Validater
+		$this->validate($request, $this->_createRules());
```

Comment:
5.5 version, It retun request data from validation.
`$data = $this->validate(request(), rules())`

Reply:
Từ laravel 5.5. Validation nó return về cả data cho mình đó ?